libstuff/STCPServer.cpp

```diff
@@ -23,6 +23,17 @@ STCPServer::Port* STCPServer::openPort(const string& host) {
     return &*portIt;
 }
 
+
+void STCPServer::shutdownPorts() {
```

Comment:
What is the difference between this and `STCPServer::closePorts()`?  Can you even shut down a port?